src/Microsoft.AspNetCore.Http.Abstractions/HttpMethods.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Http
+{
+    public static class HttpMethods
```

Comment:
Should this be singular?
`HttpMethod.Get`
`HttpMethod.IsGet(string)`
`HttpMethod.Equals(string, string)`